exporter/stackdriverexporter/stackdriverexporter_test.go

```diff
@@ -14,4 +14,124 @@
 
 package stackdriverexporter
 
-// TODO: Add tests.
+import (
+	"context"
+	"reflect"
+	"testing"
+	"time"
+
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	commonpb "github.com/census-instrumentation/opencensus-proto/gen-go/agent/common/v1"
+	metricspb "github.com/census-instrumentation/opencensus-proto/gen-go/metrics/v1"
+	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
+	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+	"github.com/census-instrumentation/opencensus-service/data"
```

Comment:
nit: blank line to separate the import block. Also the repo itself should be the last "block", so the 3 last ones need to be moved to the 2nd block.

Reply:
Fixed